packages/cache/localforage/src/index.ts

```diff
@@ -13,7 +13,6 @@ export default class LocalforageCache<V = any> implements KeyValueCache<V> {
     this.localforage = LocalForage.createInstance({
       name: config?.name || 'graphql-mesh-cache',
       storeName: config?.storeName || 'graphql-mesh-cache-store',
-      // @ts-expect-error - Weird error
       driver: driverNames.map(driverName => LocalForage[driverName] ?? driverName),
```

Comment:
We let this like that ? Or we should create an issue to fix it later ?

Reply:
There is nothing to fix. What do you mean?